packages/machine/src/ethereum/utils/free-balance.ts

```diff
@@ -0,0 +1,69 @@
+import ETHBucket from "@counterfactual/contracts/build/contracts/ETHBucket.json";
+import { AppInterface, ETHBucketAppState } from "@counterfactual/types";
+import { AddressZero, MaxUint256 } from "ethers/constants";
+import {
+  defaultAbiCoder,
+  // formatParamType,
```

Comment:
remove?

Reply:
I want to keep it here just because there is a big `TODO` further down in the code that will dramatically benefit from the usage of this and I think it's useful to keep in mind. Maybe unnecessary though and that should just be an issue instead..